### DISCUSSION TITLE
Lock/Unlock

### DISCUSSION THREAD

**Discussion by atulpg** [Jyers/Marlin#659] [Ideas]
Hello Guys.. Am running the latest jyers firmware E3V2-Default-v4.2.7.bin and very happy with it. I had a request how can I add 2 functions within the firmware as below -
After giving the print command, can we add a lock/unlock screen option ? My daughter keeps on playing with it and there are times she has just stopped the prints, paused them.
Once the print is finished.. can we add a random beeping sound to denote that the print is finished.

**Comment by Cal-LF** [comment]
There is a screen off function in the tune menu. Teach young lady to not mess with daddy's stuff.
